src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days < 3) {
+    return totalCost;
```

Comment:
Use an `if` with `return` and remove the `else` to simplify the code structure. According to checklist item 5, you don't need `else` after an `if` that contains a `return` statement.